src/components/Panel/Panel.less

```diff
@@ -6,23 +6,36 @@
 	background-color: @color-white;
 	border: @border-standardBorder;
 
-	&-has-margin {
+	.lucid-Panel-has-margin& {
 		margin: 0 @size-standard @size-standard @size-standard;
 	}
+	
+	.lucid-Panel-is-scrollable& {
+		display: flex;
+		flex-direction: column;
+	}
 
-	&-Header {
+	& > &-Header {
 		padding: @size-standard;
 		border-bottom: @border-lightBorder;
 	}
 
-	&-is-not-gutterless &-content {
-		padding: @size-standard;
-		overflow: auto;
+	& > &-content {
+		.lucid-Panel-is-not-gutterless& {
```

Comment:
This is following the LESS docs to select for parent classes (http://lesscss.org/features/#parent-selectors-feature-changing-selector-order).

For example we could have three different selectors with `&-content` in three different places, which resembles  vanilla CSS more closely:

```
& > &-content {
  //default styles
}
&&-is-not-gutterless > &-content {...}
&&-is-scrollable > &-content {...}
```

Or, we can have one section for content with defaults styles with a parent class selector to extend them:

```
& > &-content {
  //default content styles

  .lucid-Panel-is-not-gutterless& {...}
  .lucid-Panel-is-scrollable& {...}
}
```

Both approaches produce the same CSS, but the latter is better for codeorg by making use of available LESS features. I've used this pattern in other components in this repo, and feel we should be making more use of it.